docs/docs/userguide/CONFIGURATION.md

```diff
@@ -338,7 +338,15 @@ desired_capacity = <NUMBER OF NODES>  # between 1 and 10
     confluence_min_heap            = "256m"
     confluence_max_heap            = "512m"
     ```
+    
+    ### Collaborative editing
+
+    `confluence_enable_synchrony` enables [Collaborative editing](https://confluence.atlassian.com/doc/collaborative-editing-858771779.html). (default: `true`)
 
+    ```terraform
+    confluence_enable_synchrony = true
```

Comment:
When I was adding the property, shouldn't we call it `confluence_enabled_collaborative_editing`? I am not sure if Synchrony is a commonly known term.

Reply:
I agree I think `confluence_enabled_collaborative_editing` is a better fit

Reply:
I think even `confluence_collaborative-editing_enabled` is better option.

Reply:
I have renamed the property to `confluence_collaborative_editing_enabled`.